FreydCategoriesForCAP/gap/RingsAsAbCats.gi

```diff
@@ -302,6 +312,109 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_RING_AS_CATEGORY,
             
         end );
         
+        ##
+        AddTensorUnit( category,
+          function( cat )
+            
+            return RingAsCategoryUniqueObject( cat );
+            
+        end );
+        
+        ##
+        AddTensorProductOnObjects( category,
+          function( cat, a, b )
+            
+            return TensorUnit( category );
+            
+        end );
+        
+        ##
+        AddTensorProductOnMorphismsWithGivenTensorProducts( category,
+          function( cat, source, alpha, beta, range )
+            
+            return MorphismConstructor( cat,
+                           source,
+                           UnderlyingRingElement( alpha ) * UnderlyingRingElement( beta ),
+                           range );
+            
+        end );
+        
+        ##
+        AddBraidingWithGivenTensorProducts( category,
+          function( cat, object_1_tensored_object_2, object_1, object_2, object_2_tensored_object_1 )
+            
+            return IdentityMorphism( cat,
+                           TensorUnit( cat ) );
+            
+        end );
+        
+        ##
+        AddDualOnObjects( category, { cat, obj } -> obj );
+        
+        ##
+        AddDualOnMorphismsWithGivenDuals( category,
+          function( cat, dual_source, morphism, dual_range )
+            
+            return morphism;
+            
+        end );
+        
+        ##
+        AddEvaluationForDualWithGivenTensorProduct( category,
+          function( cat, tensor_object, object, unit )
+            
+            return IdentityMorphism( cat,
+                           TensorUnit( cat ) );
+            
+        end );
+        
+        ##
+        AddCoevaluationForDualWithGivenTensorProduct( category,
+          function( cat, unit, object, tensor_object )
+            
+            return IdentityMorphism( cat,
+                           TensorUnit( cat ) );
+            
+        end );
+        
+        ##
+        AddMorphismToBidualWithGivenBidual( category, { cat, obj, dual } -> IdentityMorphism( cat, obj ) );
```

Comment:
This can be derived, so I suggest to not install it.

Reply:
Done.